### PR TITLE
Event: Hackathon "Imaging in Galaxy", Freiburg 2024

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
Listing and advertising the upcoming hackathon in Freiburg next month. Information collected from registration page and the event's [Google Doc](https://docs.google.com/document/d/1pVToHXfwDLCkRZF0ifg9aVeT3cg-cjGEYWVSyIRd2hc/edit).

@beatrizserrano, please check

### CODE REVIEW

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:21]: can we remove this line so that we don't receive much spam? it's already in the form :)

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:43]: I'd also avoid publishing this, it's on the agenda. We can link the [agenda](https://docs.google.com/document/d/1pVToHXfwDLCkRZF0ifg9aVeT3cg-cjGEYWVSyIRd2hc/edit#heading=h.84pge7bu74zm) somewhere

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:29]:
```suggestion
Registration is free, but the space is limited, so we recommend to register soon. Please wait for our confirmation before booking your trip.

If you have trouble covering for travel and accommodation, please contact us directly to discuss options.
```

Comment (beatrizserrano) [reply content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:29]: this suggestion came from the NFDI office

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:77]: we are waiting for the confirmation from NFDI that this is the right statement

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:34]:
```suggestion
  <a class="btn w-25 btn-dark" href="https://forms.gle/A2VBPtFwVJjpB4tc7" role="button" target="_blank">Register now</a>
```

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:18]:
```suggestion
- **Location**: Seminar room at Georges-Köhler-Allee 79, 79110 Freiburg
```

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:16]:
```suggestion
- **Date**: Feb 26th - Mar 1st 2024 (full week)
```

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:31]:
```suggestion
As this is a hybrid event, remote attendance is also possible (details in the registration).
```

File: content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md
Comment (beatrizserrano) [inline content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md:70]: ❤️

**Review by beatrizserrano — COMMENTED** [review]
Looks good to me, thanks @sebastian-schaaf. Only some minor suggestions. Let's wait until next week to merge when we have some answers from NFDI. I'll edit here when I have them and merge. Danke!

**Comment by sebastian-schaaf** [comment]
Thanks @beatrizserrano - I applied your suggestions, now we are finally waiting for NFDI's response I guess.

**Review by bgruening — APPROVED** [review]

**Comment by beatrizserrano** [comment]
Thanks so much @sebastian-schaaf for working on this! However, this should have not been merged cause we are pending the proper GDPR statement. We'll have to open another PR fixing this.

**Comment by bgruening** [comment]
What GDPR statement? Who is providing this? Do we need to ask Christian?

**Comment by beatrizserrano** [comment]
> What GDPR statement? Who is providing this? Do we need to ask Christian?

read your email from Christian from yesterday morning ;)

**Comment by bgruening** [comment]
So this might help? https://uni-freiburg.de/en/data-protection-applications/
I'm not sure why this is to be added here. We usually put this into the registration form.

**Comment by sebastian-schaaf** [comment]
Ok, that one is much shorter (and less precise) than the one I found:
https://uni-freiburg.de/zuv/datenschutzerklaerung/

This is in German and (by title) not restricted to applications. However, the one linked by you should do the job, not least brings less obstacles.

**Pull request merged**
